include/odp/api/spec/classification.h

```diff
@@ -164,9 +171,40 @@ typedef enum {
  * Used to communicate class of service creation options
  */
 typedef struct odp_cls_cos_param {
-	odp_queue_t queue;	/**< Queue associated with CoS */
-	odp_pool_t pool;	/**< Pool associated with CoS */
-	odp_cls_drop_t drop_policy;	/**< Drop policy associated with CoS */
+	/** Number of queues to be linked to this CoS.
+	 * If the number is greater than 1 then hashing is enabled.
+	 * If number is equal to 1 then hashing is disabled.
+	 * When hashing is enabled the queues are created by the implementation
+	 * and application need not configure any queue to the class of service.
+	 * When hashing is disabled application has to configure the queue to
+	 * the class of service.
+	 * Depening on the implementation this number might be rounded-off to
+	 * nearest supported value (e.g power of 2)
+	 */
+	uint32_t num_queue;
+
+	union {
```

Comment:
Doxygen wants a comment for this union. Every struct/union/var needs some doxygen comment. Suggestion:

/** Variant mappings for queue hash configuration */